x-pack/plugins/monitoring/server/lib/alerts/fetch_ccr_read_exceptions.test.ts

```diff
@@ -47,9 +47,32 @@ describe('fetchCCReadExceptions', () => {
           bool: {
             filter: [
               {
-                nested: {
-                  path: 'ccr_stats.read_exceptions',
-                  query: { exists: { field: 'ccr_stats.read_exceptions.exception' } },
+                bool: {
+                  should: [
+                    {
+                      nested: {
```

Comment:
since nested fields can't be aliased when they don't belong to the same scope, these paths would throw an error when one mapping doesnt exist, so ignoring

Reply:
That's how the logstash queries were designed as well so we're good as far as consistency goes :)